Core/Utilities/MySQL.py

```diff
@@ -428,11 +428,11 @@ def _except(self, methodName, x, err):
     try:
       raise x
     except MySQLdb.Error as e:
-      self.log.debug('%s: %s' % (methodName, err),
+      #self.log.debug('%s: %s' % (methodName, err),
                      '%d: %s' % (e.args[0], e.args[1]))
```

Comment:
Also this line needs to be commented out (https://travis-ci.org/DIRACGrid/DIRAC/jobs/598620679#L813)

And also please take care of the format: https://travis-ci.org/DIRACGrid/DIRAC/jobs/598620680

Reply:
yes, thanks!